File: app/src/main/java/com/german/keyboard/app/free/changes/Premium.java
package com.german.keyboard.app.free.changes;

import android.os.Bundle;
import android.view.MenuItem;
import android.view.View;

import androidx.annotation.NonNull;

import com.akexorcist.localizationactivity.ui.LocalizationActivity;
import com.module.ads.AddInitilizer;

import java.util.Objects;

import com.german.keyboard.app.free.R;

public class Premium extends LocalizationActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_premium);
        try{
            getSupportActionBar().hide();
        }catch (Exception e){
            e.printStackTrace();
        }


        Objects.requireNonNull(getSupportActionBar()).setDisplayHomeAsUpEnabled(true);
        findViewById(R.id.backBtn).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });
        findViewById(R.id.go_premium).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                new AddInitilizer(getApplicationContext(),Premium.this,null).goAddFree();
            }
        });

    }

    @Override
    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
        switch (item.getItemId()) {
            case android.R.id.home:
                this.finish();
                return true;
        }
        return super.onOptionsItemSelected(item);
    }
}

File: app/src/main/java/com/german/keyboard/app/free/changes/Premium.kt
package com.german.keyboard.app.free.changes

import android.annotation.SuppressLint
import com.akexorcist.localizationactivity.ui.LocalizationActivity
import android.os.Bundle
import android.view.MenuItem
import android.view.View
import com.german.keyboard.app.free.R
import com.module.ads.AddInitilizer
import java.lang.Exception
import java.util.*

class Premium : LocalizationActivity() {
    @SuppressLint("MissingSuperCall")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_premium)
        try {
            supportActionBar!!.hide()
        } catch (e: Exception) {
            e.printStackTrace()
        }
        Objects.requireNonNull(supportActionBar)!!.setDisplayHomeAsUpEnabled(true)
        findViewById<View>(R.id.backBtn).setOnClickListener { finish() }
        findViewById<View>(R.id.go_premium).setOnClickListener {
            AddInitilizer(
                applicationContext,
                this@Premium,
                null
            ).goAddFree()
        }
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when (item.itemId) {
            android.R.id.home -> {
                finish()
                return true
            }
        }
        return super.onOptionsItemSelected(item)
    }
}